assets/schema/ensemble_schema.json

```diff
@@ -2898,6 +2898,12 @@
                       "type": "integer",
                       "description": "The initial page to show when first creating the Carousel and the current index of the displaying Carousel Item. Default (0)"
                     },
+                    "selectedItemIndex": {
+                      "type": "integer",
+                      "description": "The initial page to show when first creating the Carousel and the current index of the displaying Carousel Item. Default (0)",
+                      "deprecated": false,
```

Comment:
False? correct me if I am wrong it is supposed to be a deprecated

Reply:
Yes, but not now. So I added it as false and with deprecation message

Reply:
@snehmehta I updated it.